### ISSUE TITLE
Truly lossless by default?

### ISSUE THREAD

**Issue by gamax92** [shssoichiro/oxipng#164]
Glad to see that a replacement for optipng exists, but a few things in the usage info have me concerned that oxipng is only perceptually lossless and might trash alpha pixels by default. If so is there any option I may use to prevent any alpha optimizations at all? In my use case, optimizing alpha will lead to black outlines around pixels. Otherwise I cannot safely use this tool.

**Comment by shssoichiro** [comment]
You are correct, right now if you use -o4 or higher, alpha optimizations will be turned on, and there is no option to force disable them. I will add this in as a new feature request.

**Comment by drojf** [comment]
I guess this is somewhat not related to this issue, but personally I would expect oxipng to not apply any alpha optimizations by default, if it is advertised as a lossless png optimizer.

I would either:

1. Not apply any alpha optimization by default
2. Put a disclaimer on the Readme.md page that the default setting removes color from zero-alpha regions (right now, alpha is not mentioned or documented on the page)

I don't know if you're familiar with cases when zero-alpha colors are important, here are two:
 - filtered textures: http://www.adriancourreges.com/blog/2017/05/09/beware-of-transparent-pixels/   
 - pre multiplied alpha https://microsoft.github.io/Win2D/html/PremultipliedAlpha.htm (note how the source's alpha is ignored)
